pom.xml

```diff
@@ -44,8 +44,8 @@ THE SOFTWARE.
 
   <properties>
     <jenkins.version>1.580.1</jenkins.version>
-    <java.level>6</java.level>
-    <mavenInterceptorsVersion>1.7</mavenInterceptorsVersion>  
+    <java.level>7</java.level>
```

Comment:
@maximusfloydus I just merged https://github.com/jenkinsci/maven-interceptors/pull/9 I need to apply few more changes on it to release it (this week). After this I'll be able to use it here and to merge this PR